vizro-core/tests/unit/vizro/conftest.py

```diff
@@ -60,11 +60,3 @@ def vizro_app():
     app instantiation.pages.
     """
     return Vizro()
-
```

Comment:
I've wanted to move this to the navigation conftest for a while, and now that I've removed all its other uses from test_dashboard.py it has been set free.